wurst/systems/units/buildings/CraftMaster.wurst

```diff
@@ -22,7 +22,7 @@ public class CraftMaster extends Building
         getUnit().addAbility(SPELL_QM_STORAGE_HUT_KIT)
         getUnit().addAbility(SPELL_QM_HATCHERY_KIT)
         getUnit().addAbility(SPELL_QM_MUD_HUT_KIT)
-        getUnit().addAbility(SPELL_QM_ARMORY_KIT)
+        getUnit().addAbility(SPELL_QM_FORGE_KIT)
```

Comment:
Is the decision here to just not enforce that an armory can be created at the craft master? That seems fine because any troll can make the armory themself, just wanted to confirm.

Reply:
Yeah thas right